### PR TITLE
Run airseeker locally

### PR DESCRIPTION
**Pull request by Siegrift**
Closes https://github.com/api3dao/airseeker-v2/issues/63

## Rationale

To run Airseeker locally, we need to start multiple services and use correct configurations for each. I wanted to preserve this integration work so I put everything inside a single folder and documented the intentions and steps in a `README.md`. 

There were a few Airseeker issues that needed to be patched. They are in a separate PR https://github.com/api3dao/airseeker-v2/pull/88

At first I though I will just open the PR for discussion and then close it, but I think we might even merge it. The files are **not** included in the build artefacts (because we use a separate tsconfig for that).

### CODE REVIEW

File: local-test-configuration/README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# Local Airseeker test
+
+The idea was to use the Docker images for Pusher, Signed API and Airseeker and run them locally. Specifically, I opted
+for the following setup:
+
+- Use 2 pushers with Nodary API and reasonable fetch limits. Each pusher has a different Airnode mnemonic to mimick a
```
Comment (aquarat) [inline local-test-configuration/README.md:6]:
```suggestion
- Use 2 pushers with Nodary API and reasonable fetch limits. Each pusher has a different Airnode mnemonic to mimic a
```

File: local-test-configuration/README.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  different API. One of the APIs is delayed so that the beacons have different values.
+- Use 2 signed APIs and each Pusher pushes to a separate Signed API.
+- We run Airseeker only on Hardhat network for setup simplicity. Initially, I wanted to have a Polygon testnet as well,
+  but gave up on that idea for now.
+
+All configurations are based on the example files, but have been slightly modified. I had to also choose a more volatile
+assets from Nodary API to see Airseeker updates.
```
Comment (aquarat) [inline local-test-configuration/README.md:13]:
```suggestion
All configurations are based on the example files, but have been slightly modified. I had to also choose more volatile
assets from the Nodary API to see Airseeker updates.
```

File: local-test-configuration/README.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+```
+
+- Deploy the test contracts and fund respective Airseeker sponsor wallets:
+
+```sh
+pnpm ts-node -T ./local-test-configuration/scripts/initialize-chain.ts
+```
+
+This command gives you the addresses of the deployed contracts. These need to be put to Airseeker secrets and are
+required also for the monitoring page.
```
Comment (aquarat) [inline local-test-configuration/README.md:64]:
```suggestion
This command gives you the addresses of the deployed contracts. These need to be put into the Airseeker secrets and are also
required for the monitoring page.
```

File: local-test-configuration/README.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+```sh
+docker run -it --init --volume $(pwd)/local-test-configuration/airseeker:/app/config --network host --env-file .env --rm api3/airseeker-v2:latest
+```
+
+## Final notes
+
+- Monitor the logs of each service, pay attention to errors, warnings and possible improvements.
+- Use the monitoring page to see if the feeds update when they should.
+- I also tried changing the batch size in Airseeker to 1 and Airseeker worked as expected (but was more chattier).
```
Comment (aquarat) [inline local-test-configuration/README.md:86]:
```suggestion
- I also tried changing the batch size in Airseeker to 1 and Airseeker worked as expected (but was chattier).
```

**Review by aquarat — APPROVED** [review]
LGTM, very cool @ monitoring HTML page and the local-test-configuration README :)

File: local-test-configuration/README.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
+# Local Airseeker test
+
+The idea was to use the Docker images for Pusher, Signed API and Airseeker and run them locally. Specifically, I opted
+for the following setup:
```
[the remark this answers is not in the document]
Comment (Siegrift) [reply local-test-configuration/README.md:4]: Good point.

File: local-test-configuration/README.md (hunk 4 above)
Comment (Siegrift) [reply local-test-configuration/README.md:86]: Even better is to avoid the first person sentence.

**Pull request merged**